tools/icqsol_add_texture/icqsol_add_texture.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+import argparse
+import shutil
+from pkg_resources import resource_filename
```

Comment:
Greg,
This is the only and recommended way to access data files. We're already assuming setuptools installed and pkg_resources comes with it. There is no additional installation. 

--Alex